apps/web-giddh/src/app/vouchers/create/create.component.ts

```diff
@@ -1233,9 +1243,14 @@ export class VoucherCreateComponent implements OnInit, OnDestroy, AfterViewInit
      * @private
      * @memberof VoucherCreateComponent
      */
-    private getVoucherType(): void {
-        this.invoiceType = this.vouchersUtilityService.getVoucherType(this.voucherType);
-        this.currentVoucherFormDetails = this.vouchersUtilityService.prepareVoucherForm(this.voucherType);
+    private getVoucherType(response?: any): void {
+        if (response) {
+            let isCashInvoice = response.cashVoucher ? true : false;
```

Comment:
Voucher type can be detected from url

Reply:
We are not dependent voucher type in update mode, we are using cashVoucher from response, for create we are using by default